receiver/aerospikereceiver/go.mod

```diff
@@ -0,0 +1,53 @@
+module github.com/open-telemetry/opentelemetry-collector-contrib/receiver/aerospikereceiver
+
+go 1.17
+
+require (
+	github.com/aerospike/aerospike-client-go/v5 v5.8.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/internal/containertest v0.50.0
```

Comment:
It looks like there's some discussion about this, no action to take, just want to call this out: https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/9939